EntityFramework.Reverse.POCO.Generator/EF.Reverse.POCO.Core.ttinclude

```diff
@@ -1165,10 +1165,11 @@
                         break;
 
                     case "datetime":
+                    case "datetime2":
```

Comment:
There is no datetime2 on system or am I miss understanding of what system.* is used for?